packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts

```diff
@@ -50,29 +49,27 @@ import {
   ResourcePluginsV2,
 } from "../ResourcePluginContainer";
 import { checkWetherProvisionSucceeded, getSelectedPlugins, isAzureProject } from "./utils";
+import { isV3 } from "../../../..";
 
 export async function getQuestionsForScaffolding(
   ctx: v2.Context,
   inputs: Inputs
-): Promise<Result<QTreeNode | undefined, FxError>> {
-  const node = new QTreeNode({ type: "group" });
-
-  // 1. capabilities
-  const capQuestion = createCapabilityQuestion();
-  const capNode = new QTreeNode(capQuestion);
-  node.addChild(capNode);
+): Promise<Result<QTreeNode | QTreeNode[] | undefined, FxError>> {
+  const node: QTreeNode[] = [];
 
-  // 1.1.1 SPFX Tab
-  const spfxPlugin: v2.ResourcePlugin = Container.get<v2.ResourcePlugin>(
-    ResourcePluginsV2.SpfxPlugin
-  );
-  if (spfxPlugin.getQuestionsForScaffolding) {
-    const res = await spfxPlugin.getQuestionsForScaffolding(ctx, inputs);
-    if (res.isErr()) return res;
-    if (res.value) {
-      const spfxNode = res.value as QTreeNode;
-      spfxNode.condition = { contains: TabSPFxItem.id };
-      if (spfxNode.data) capNode.addChild(spfxNode);
+  if (!isV3()) {
```

Comment:
For V2: only TeamsAppSolution in the list and spfx is included
For V3: TeamsAppSolution + TeamsSPFxSolution in the list, and tab+bot+me in TeamsAppSolution, spfx in TeamsSPFxSolution